src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -177,6 +176,7 @@ const NetworkModificationNodeEditor = () => {
     const [messageId, setMessageId] = useState('');
     const [launchLoader, setLaunchLoader] = useState(false);
     const [isUpdate, setIsUpdate] = useState(false);
+    const [anchorEl, setAnchorEl] = useState(null);
```

Comment:
You can set this with ref=anchorEl. Here, you declare : const buttonAddRef = useRef();